features/division.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Division

```

Comment:
No divide by zero scenario. You have the (correct) steps defined, so I'm guessing it was an oversight.